src/main/webapp/dev/js/common/questionMcq.js

```diff
@@ -80,10 +83,31 @@ function toggleMcqOtherOptionEnabled(checkbox, questionNum) {
         $(questionId).attr('editStatus', 'mustDeleteResponses');
     }
 }
+/**
+ * Enables MCQ options for a question to be reordered through a drag and drop mechanism.
+ * Binds an update event to the option elements which is triggered whenever the order of
+ * elements change. The event handler updates the ids of elements to match the new order.
+ */
+function makeMcqOptionsReorderable(questionNum) {
+    $(`#mcqOptionRows-${questionNum}`).sortable({
+        cursor: 'move',
+        update() {
```

Comment:
```
update: function() {
    //the function
}
```

Reply:
i was using that earlier but it gave the lint error `Expected method shorthand  object-shorthand` so I changed it